kratos/includes/variables.h

```diff
@@ -97,6 +97,8 @@ namespace Kratos
     KRATOS_DEFINE_VARIABLE( double, POSITIVE_FACE_PRESSURE )
     KRATOS_DEFINE_VARIABLE( double, FACE_HEAT_FLUX )
 
+    KRATOS_DEFINE_VARIABLE( double, PD )
```

Comment:
A more descriptive name?

Reply:
removed and moved to OptimizationApplication